File: Coding/Java/Set4/FizzBuzz.java
package Set4;

public class FizzBuzz {
    public static String fb(int num) {
        if (num % 3 == 0 && num % 5 == 0) {
            return "FizzBuzz";
        } else if (num % 3 == 0) {
            return "Fizz";
        } else if (num % 5 == 0) {
            return "Buzz";
        }
        return String.valueOf(num);
    }
    public static void main(String[] args) {
        System.out.println(fb(15));
        System.out.println(fb(3));
        System.out.println(fb(5));
        System.out.println(fb(2));
    }
}

File: Coding/JavaScript/Set4/FizzBuzz.js
function fizzBuzz(num){
    if(num % 5== 0 && num%3==0){
        return "FizzBuzz"
    } else if(num % 3 == 0){
        return "Fizz"
    } else if(num % 5 == 0){
        return "Buzz"
    } else {
        return num.toString()
    }
}

console.log(fizzBuzz(15))
console.log(fizzBuzz(3))
console.log(fizzBuzz(5))
console.log(fizzBuzz(2))